{
    "type": "bundle",
    "id": "bundle--049f174c-6221-49f4-9618-94672a7d71a5",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--98ad045b-e632-49ca-b16d-c7e58fbf009d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-09-29T00:21:39.004441Z",
            "modified": "2021-09-29T00:21:39.004441Z",
            "name": "CVE-2021-36365",
            "description": "Nagios XI before 5.8.5 has Incorrect Permission Assignment for repairmysql.sh.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2021-36365"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}